addons/website_slides/models/slide_channel.py

```diff
@@ -477,6 +471,12 @@ def _compute_website_background_image_url(self):
                      % ('training' if channel.channel_type == 'training' else 'documentation')
             )
 
+    def _get_placeholder_filename(self, field):
```

Comment:
With this rewrite,
* You can now call it in slide.slide as was previously present
* _compute_website_background_image_url needs to be updated (and renamed?), because we'll only be calling it to get the default image (when we know there is no image_1920.